frontend/components/matching/matching-form.tsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import { Button } from "@/components/ui/button";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { Label } from "@/components/ui/label";
+
+const difficulties: string[] = ["Easy", "Medium", "Hard"];
```

Comment:
Might be best to add request to get possible values of difficulties, as well as topics. Though, for topics, we should wait till #159 is done. For the purposes of this PR, I think this is fine for now.

Reply:
I think better to do in another PR